libs/service-portal/occupational-licenses/src/components/LicenseDetail.tsx

```diff
@@ -107,11 +108,29 @@ export const LicenseDetail: React.FC<LicenseDetailProps> = ({
                 columnGap="p1"
               >
                 <Text>
-                  {formatMessage(isValid ? om.validLicense : om.invalidLicense)}
+                  {formatMessage(
+                    isValid === 'valid'
+                      ? om.validLicense
+                      : isValid === 'limited'
+                      ? om.validWithLimitationsLicense
+                      : om.invalidLicense,
+                  )}
                 </Text>
                 <Icon
-                  icon={isValid ? 'checkmarkCircle' : 'closeCircle'}
-                  color={isValid ? 'mint600' : 'red600'}
+                  icon={
+                    isValid === 'valid'
+                      ? 'checkmarkCircle'
+                      : isValid === 'limited'
+                      ? 'warning'
+                      : 'closeCircle'
+                  }
+                  color={
+                    isValid === 'valid'
+                      ? 'mint600'
+                      : isValid === 'limited'
+                      ? 'yellow600'
+                      : 'red600'
```

Comment:
Not consistent with `LicenceActionCard.tsx` that has

```
        variant:
          isValid === 'valid'
            ? 'blue'
            : isValid === 'limited'
            ? 'yellow'
            : 'rose',
``` 

is that on purpose?